utils.ps1

```diff
@@ -0,0 +1,298 @@
+# Need this variable as long as we support PS v2
+$ModuleBasePath = Split-Path $MyInvocation.MyCommand.Path -Parent
+
```

Comment:
Correct, in addition, we do not support v2 - dbatools is v3 and up, thankfully ;)